backend/tests/test_project.py

```diff
@@ -99,15 +196,15 @@ def test_get_project_by_id(self):
         )
         p = Project(
             id=p_id,
+            type="dummy type",
+            contacts=[c],
```

Comment:
@peterghrong minor issue, usually for places in the code where you made accidental changes, you would want to revert it back to how it was originally before. So there should not be any line changes in this part.

Reply:
Will do next time